aptos-move/framework/src/natives/aggregator_natives/context.rs

```diff
@@ -213,4 +243,22 @@ mod test {
         assert!(changes.contains_key(&aggregator_id_for_test(700)));
```

Comment:
Can you modify this test to check it's a delta here? and also for key 600? Then yin your new test we can check if it became a write

Reply:
Done.